astroquery/esa/xmm_newton/core.py

```diff
@@ -11,19 +11,11 @@
 
 
 """
-import os
 import re
-from astroquery.utils import commons
-from astropy import units
-from astropy.units import Quantity
-from astroquery.utils.tap.core import TapPlus
-from astroquery.utils.tap.model import modelutils
-from astroquery.query import BaseQuery
-from astropy.table import Table
-from requests.exceptions import HTTPError
+from ...utils.tap.core import TapPlus
```

Comment:
absolute imports should work, but making that change can be part of a bigger, package wide cleanup.